### ISSUE TITLE
Why labels in generator and discriminator train are both np.zeros(BATCHSIZE, 1)?

### ISSUE THREAD

**Issue by oubathAI** [IShengFang/Relativistic-average-GAN-Keras#2]
Hi
Thanks for your implementation of RaGAN with Keras.
There is just one thing I didn't fully understood; in  RaGAN_with_Custom_Loss, I didn't get why you used labels `dummy_y = np.zeros((BATCHSIZE, 1), dtype=np.float32)` for both discriminator and generator train.
These labels corresponding to Discriminator_real_out, shouldn't be `np.ones` instead?

Thanks for your reply!

**Comment by IShengFang** [comment]
Hi @oubathAI ,

Because every loss compiled in model must has its target tensor.
Check out line 139 and 145.[1]
``` python
generator_train.compile(OPT, loss=[relativistic_generator_loss, None])
discriminator_train.compile(OPT, loss=[relativistic_discriminator_loss, None])
```
I compile model with only one loss.
Also , the labels of real or fake digit is directly implement in the custom loss.
Check out line 105-135.[2]

ref:
[1]
https://github.com/IShengFang/Relativistic-average-GAN-Keras/blob/7a816e1872b3cb8afa4344e25860ce3a59c4485c/RaGAN_CustomLoss.py#L137-L146
[2]
https://github.com/IShengFang/Relativistic-average-GAN-Keras/blob/7a816e1872b3cb8afa4344e25860ce3a59c4485c/RaGAN_CustomLoss.py#L105-L135

Sorry for my late reply.
